contracts/dispatcher/src/contract.rs

```diff
@@ -83,8 +83,8 @@ pub fn execute(
 ) -> ContractResult<CwResponse> {
     match msg {
         ExecuteMsg::TimeAlarm {} => {
-            SingleUserAccess::load(deps.storage, crate::access_control::TIMEALARMS_NAMESPACE)?
-                .check_access(&info.sender)?;
+            SingleUserAccess::new(&*deps.storage, crate::access_control::TIMEALARMS_NAMESPACE)
```

Comment:
Just to clear things up, which one do we prefer because this was something you mentioned in previous PR? `&*` or `.deref()`?

Reply:
ooops, as we agreed before, we stick to the more explanatory `.deref()`